README.md

```diff
@@ -74,11 +74,19 @@ you'll also need to specify the `+crt-static` target feature.
 RUSTFLAGS="-C target-feature=+crt-static -C linker=aarch64-linux-gnu-gcc" cargo build --target aarch64-unknown-linux-gnu
 ```
 
-This will require installing the appropriate cross-platform compiler and linker
-toolchain for your target platform. Examples of how we cross-compile rav1d in CI
-can be found in
+This will require installing the `rustup` component for the target platform and
+the appropriate cross-platform compiler/linker toolchain for your target
+platform. Examples of how we cross-compile rav1d in CI can be found in
 [`.github/workflows/build-and-test-qemu.yml`](.github/workflows/build-and-test-qemu.yml).
 
+The following targets are currently supported:
+
+* `x86_64-unknown-linux-gnu`
+* `i686-unknown-linux-gnu`
+* `armv7-unknown-linux-gnueabihf`
+* `aarch64-unknown-linux-gnu`
+* `riscv64gc-unknown-linux-gnu`
```

Comment:
Do we support any `target_arch = "riscv32"` targets?  Because we have `#[cfg]`s for that `target_arch`, but I'm not sure which targets it's meant to correspond to.

Reply:
We don't test it in CI so I don't think we want to list it here for now.

Reply:
Yeah, I saw https://github.com/memorysafety/rav1d/pull/1182#discussion_r1663741892 about it, too; makes sense.